packages/account-v2/src/utils/connectedAppsUtils.ts

```diff
@@ -0,0 +1,15 @@
+type TConnectedAppsHeader = {
+    [key: string]: string;
+};
+
+export const connectedAppsHeader: TConnectedAppsHeader = {
+    action: 'Action',
+    lastLogin: 'Last Login',
+    name: 'Name',
+    permission: 'Permission',
+};
+
+export const connectedAppsHeaderOrder = ['name', 'permission', 'lastLogin', 'action'];
```

Comment:
Would be nice to use uppercase naming for these variables to follow the standards.
https://github.com/airbnb/javascript?tab=readme-ov-file#naming--uppercase

Reply:
Instead I could change `connectedAppsHeader` to uppercase? since, its a const 🙏